package.json

```diff
@@ -4,7 +4,8 @@
   "description": "PublicLab.Editor is a general purpose, JavaScript/Bootstrap UI framework for rich text posting, which provides an author-friendly, minimal, mobile/desktop (fluid) interface for creating blog-like content, designed for PublicLab.org",
   "main": "dist/PublicLab.Editor.js",
   "scripts": {
-    "build": "grunt build",
+    "build": "grunt build && ./node_modules/.bin/babel dist -d dist",
```

Comment:
```suggestion
    "build": "grunt build && npm run babel",
```

Reply:
This looks great, aren't we bypassing the need for Grunt to understand babel by making npm able to run babel?